src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,28 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0
+
+* Added support for connecting clusters that use the OVNKubernetes CNI plugin.
+* The active Gateway now performs periodic health checks on the connections to remote clusters, updates the Gateway
+  connection status, and adds latency statistics.
+* Gateways now export the following connection metrics on TCP port 8080 which can be used with Prometheus. These are
+  currently only supported for the Libreswan cable driver.
+  * The count of bytes transmitted and received between Gateways.
+  * The number of connections between Gateways and their corresponding status.
+  * The timestamp of the last successful connection established between Gateways.
+  * The RTT latency between Gateways.
+* The Libreswan cable driver is now the default.
+* The StrongSwan cable driver is deprecated and will be removed in a future release.
```

Comment:
```suggestion
* The strongSwan cable driver is deprecated and will be removed in a future release.
```

Reply:
We reference other cable drivers capitalized so why not this one as well to be consistent.

Reply:
This is how it called: https://www.strongswan.org/

Reply:
yeah if we want to go by the logo name. Not a big deal - I'll change it back.